controls/scheduler/specialslots/overview.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Special Slots
+page_title: .NET MAUI Scheduler Documentation - Special Slots
+description: Learn more about the special slots feature in the Telerik UI for .NET MAUI Scheduler control.
+position: 
+slug: scheduler-special-slots
+---
+
+# Special Slots
+
+The Telerik UI for .NET MAUI Scheduler control exposes an option to define the special and read-only slots and apply different styles to them. You need to prepare a collection of Slot objects and assign it to `SpecialSlotsSource` property of the `RadScheduler` instance.
+
+Every Slot has the following properties:
+
+* `Start (type DateTime)`&mdash;Defines the start date of the slot.
```

Comment:
Be consistent with the casing of "Slots". Either "Slots" or "slots". Don't mix both.